src/index.js

```diff
@@ -38,17 +50,32 @@ const createElement = (tagName, ...options) => {
   return element;
 };
 
+
 const drawMaze = () =>
   createElement(
     "table",
     { id: "table" },
     labyrinth.map((x) =>
       createElement(
         "tr",
-        x.map((i) => createElement("td", { className: i }))
+        x.map((i) => createElement("td", { className: classNameTeg(i) }))
```

Comment:
1. I guess that there is a typo in word teg. It should be written with letter 'a'.
2. I would like to suggest consider other name function name. Like 'getClassName', 'getClassNameByToken', 'getClassNameBySymbol'

Reply:
corrected